### DISCUSSION TITLE
Language & SetAllIntl parameters

### DISCUSSION THREAD

**Discussion by Disrupi** [OSDeploy/OSD#147] [Q&A]
First of all great great project you guys have here going!

I have an issue or missunderstanding on how -Language & -SetAllIntl parameters seem to work. 
My goal is to build a ISO file which yields a device having "en-us" as OS-language but have "de-de" or preferably "de-at" for region/input-locale.
Using
New-OSDCloudTemplate -name name -Language en-us -SetAllIntl de-de
yields error 87 while building the ISO (dart v10 is installed)

It would be much appreciated!

**Comment by MrMortensen** [comment]
Hi Disrupi
You properly got on with it. I just want to answer your question, if others have the same problems.
The command "New-OSDCloudTemplate -name name -Language en-us -SetAllIntl de-de" is for the WinPE on the USB. If you want OS language to be English and German keyboard, then have a look her: https://github.com/OSDeploy/OSD/issues/197
Kind regards
Morten
